src/custom-elements/src/components/textarea/textarea.tsx

```diff
@@ -35,6 +35,10 @@ export class Textarea {
    * To define size of Textarea
    */
   @Prop({ reflect: true }) size ?: TextInputSizes = 'md';
+  /**
+   * To read value of Textarea
+   */
+  @Prop({ attribute: null, mutable: true }) value = '';
```

Comment:
You need to update this prop when the component is loaded and changed so users can read textarea value from it.